docs/dev-setup-non-vagrant.md

```diff
@@ -342,7 +342,7 @@ fi
 ./tools/do-destroy-rebuild-test-database
 ./manage.py compilemessages
 sudo ./scripts/lib/install-node
```

Comment:
It is installed in node_cache.py so should be ok.
It was originally in install_node. The problem is that yarn installs itself under the user and not globally. This is problematic because in production the user which installs node is different than the one which installs node_modules.

We could fix this by using the debian package. But that would probably make the install step longer since we would need to add an apt repository, run apt-get update etc.